### ISSUE TITLE
Allow searching 'between' values?

### ISSUE THREAD

**Issue by lurch** [resin-io/pensieve#228]
Something I've just thought of, although I'm not sure if it'll actually be useful in practice...
(if this is unlikely to be useful, then feel free to close this issue)

For those fields that are order-able (numbers, dates, semvers, etc.), as well as the ability to search for entries where "X is Y", "X is less than Y" or "X is greater than Y", perhaps it'd also be useful to search for entries where "X is between Y and Z" ?

**Comment by alexandrosm** [comment]
yeah, this would be very nice for semver-ish things too. I guess what we're
looking for is a "greater than X and less than Y" operator.

-- 

*Alexandros Marinos*

Founder & CEO, Resin.io


+1 206-637-5498

@alexandrosm
